service/matching/taskQueueManager.go

```diff
@@ -605,3 +613,12 @@ func (c *taskQueueManagerImpl) QueueID() *taskQueueID {
 func (c *taskQueueManagerImpl) TaskQueueKind() enumspb.TaskQueueKind {
 	return c.taskQueueKind
 }
+
+func (c *taskQueueManagerImpl) checkRecentPollers() bool {
+	var accessTime = time.Now().Add(-noPollerThreshold)
```

Comment:
General Go style note: when initializing with a value, prefer `:=`
```suggestion
	accessTime := time.Now().Add(-noPollerThreshold)
```

Reply:
Thanks for the tip